src/components/PaymentForm.js

```diff
@@ -7,7 +7,7 @@ class PaymentForm extends Component {
   render() {
     return (
       <Container>
-        <StripeProvider apiKey="pk_test_7rIPo7H0D768Gw8L7YdJqLAv">
+        <StripeProvider apiKey="pk_test_zxc8b5IX4U2OFSnqMAY2XN4i00tZz2JAqX">
```

Comment:
can you store this in the `.env` file

Reply:
best practice dictates that you should put in the `.env` file in the codebase and in the environment variables on netlfy.

Reply:
added to .env file, and we'll add it to netlify once it's merged with upstream